### DISCUSSION TITLE
Overriding rounded on dropdown?

### DISCUSSION THREAD

**Discussion by MGeurts** [tasteui/tasteui#578] [Q&A] [answered]
I override some properties of the dropdown:

```
TallStackUi::personalize()->dropdown()
    ->block('wrapper.third')->replace('rounded-md', 'rounded')
    ->block('action.icon')->replace('text-gray-400', 'text-primary-500 dark:text-primar-300');
```

But replacing **rounded-md** by **rounded** does not seem to work.

Any ideas?
Thanks

**Comment by devajmeireles** [comment]
This report made me understand that we have an error. You are trying to interact with a block that is no longer applied to Dropdown because we replaced `wrapper.third` with Floating, which is used in several floating components. Now we need to fix this by making it so that you can interact with the floating of this component instead of the floating for everyone. The best thing to do is create a block and let you interact with it, like `floating`. I'll work on it!

**Reply by MGeurts** [reply]
OK. Very nice. Thanks.

**Reply by devajmeireles** [reply, marked as answer]
@MGeurts https://github.com/tallstackui/tallstackui/pull/589

**Comment by devajmeireles** [comment]
Done! Released as part of v1.34.0: https://github.com/tallstackui/tallstackui/releases/tag/v1.34.0

**Comment by MGeurts** [comment]
It seems to me that overriding **rounded-md** on **wrapper.slot** on the **dropdown** is not working.

```
TallStackUi::personalize()->dropdown()
    ->block('wrapper.slot')->replace('rounded-md', 'rounded')
    ->block('width')->replace('w-56', 'w-64')
    ->block('action.icon')->replace('text-gray-400', 'text-primary-500 dark:text-primar-300');
```

Overriding on **width** and **action.icon** are working as expected.

**Reply by devajmeireles** [reply]
All components using Floating have been given a special key to handle Floating. The key is 'floating'.
